modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/js/actions.js

```diff
@@ -0,0 +1,126 @@
+/*
+ ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~      http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ */
+var jsonObj = [];
+
+function drawGraph() {
+
+    var userId = $('#UserList').val();
+
+    if (userId != '') {
+        var body = {
+            'userId': userId,
+            'order': $('#TaskDefOrder').val(),
+            'count': parseInt($('#TaskDefCount').val())
+        };
+
+        $.ajax({
+            type: 'POST',
+            url: '../../bpmn-analytics-explorer/user_level_avg_time_vs_task_id',
+            data: {'filters': JSON.stringify(body)},
+            success: function (data) {
+                var responseJsonArr = [];
+                if (!$.isEmptyObject(data)) {
+
+                    responseJsonArr = JSON.parse(data);
+
+                    var responseStr = '';
+                    for (var i = 0; i < responseJsonArr.length; i++) {
+                        var temp = '["' + responseJsonArr[i].taskDefId + '",' + responseJsonArr[i].avgExecutionTime + '],';
+                        responseStr += temp;
+                    }
+
+                    responseStr = responseStr.slice(0, -1);
+                    var jsonArrObj = JSON.parse('[' + responseStr + ']');
+                    jsonObj[0].data = jsonArrObj;
+
+                    var barChart = new vizg(jsonObj, config);
+                    barChart.draw("#chartA", [{type: "click", callback: callbackmethod}]);
+
+                }
+                else {
+                    jsonObj[0].data = [];
+                    var barChart = new vizg(jsonObj, config);
+                    barChart.draw("#chartA", [{type: "click", callback: callbackmethod}]);
+                }
+            },
+            error: function (xhr, status, error) {
+                var errorJson = eval("(" + xhr.responseText + ")");
+                alert(errorJson.message);
+            }
+        });
+    } else {
+        console.log('Empty User ID list.');
+        alert("User ID list is empty.");
```

Comment:
Both console.log and alert are not needed. You can keep alert() here. They do nearly the same.